Quick sort array

cmchan04 opened the pull request:
I have added quick sort for array. @BengRhui Take a look tomorrow bah. Take more rest ❤️

cmchan04 committed:
Include ```getMedianIndex()```

cmchan04 committed:
Quick Sort for array

BengRhui committed:
Clean up quick sort

The comparison criteria for quick sort is updated.

BengRhui commented:
@cmchan04 everything looks good on my side. Just modified the condition to compare the transaction objects.
Feel free to merge to the main branch if everything looks good on your side too.

BengRhui committed:
Add a sample file

I think you've accidentally deleted the sample file. This is added back to make sure the directory exists.

cmchan04 reviewed (commented):
ok
